core/src/main/resources/error/error-classes.json

```diff
@@ -1044,7 +1044,7 @@
   },
   "UNRESOLVED_MAP_KEY" : {
     "message" : [
-      "Cannot resolve column <objectName> as a map key. If the key is a string literal, please add single quotes around it."
+      "Cannot resolve column <objectName> as a map key. If the key is a string literal, add single quotes around it: '<object_name>'."
```

Comment:
1. I don't see that we pass `object_name`. Maybe, `objectName`?
2. We quite `objectName` by back ticks, and how it will looks like ```'`a`'```?

Reply:
Oh... I see.

Maybe we can just remove the example in the error message.

Updated PR description as well.

Reply:
> Maybe we can just remove the example in the error message

ok. Let's remove the example. You can show which quotes to add:
```
      "Cannot resolve column <objectName> as a map key. If the key is a string literal, add the single quotes '' around it."
```

Reply:
Sounds good. Just updated!